atum-s3-sdk-extension/src/main/scala/za/co/absa/atum/core/SparkQueryExecutionListenerSdkS3.scala

```diff
@@ -48,17 +48,20 @@ class SparkQueryExecutionListenerSdkS3(cf: ControlFrameworkStateSdkS3) extends S
     }
   }
 
-   /** Write _INFO file with control measurements to the output directory based on the query plan */
+  /** Write _INFO file with control measurements to the output directory based on the query plan */
   private def writeInfoFileForQueryForSdkS3(qe: QueryExecution, region: Region, kmsSettings: S3KmsSettings)(implicit credentialsProvider: AwsCredentialsProvider): Unit = {
     val infoFilePath = ExecutionPlanUtils.inferOutputInfoFileNameOnS3(qe, cf.outputInfoFileName)
 
     // Write _INFO file to the output directory
     infoFilePath.foreach(path => {
 
       import za.co.absa.atum.persistence.s3.S3LocationRegionImplicits.SimpleS3LocationRegionExt
-      import za.co.absa.atum.location.S3Location.StringS3LocationExt
+      import za.co.absa.commons.s3.S3Location._
 
-      val location = path.toS3LocationOrFail.withRegion(region)
+      val location = path.toS3Location match {
```

Comment:
Wouldn't the `S3Location.apply` be easier? No need to match.

Reply:
Sure, done.

Reply:
Commons 0.0.27 is now released.